src/App.vue

```diff
@@ -58,15 +54,8 @@ export default {
     /**
      * Connect to the event bus
      */
-    eventBus.$on('serviceWorkerUpdated', () => {
-      notifier.show({
-        message: 'New version is available',
-        type: 'confirm',
-        okText: 'Refresh',
-        cancelText: 'Close',
-        okHandler: () => window.location.reload(),
-        time: 10000,
-      });
+    eventBus.$on('serviceWorkerUpdated', () => {      
+      this.$notifyNewVersion.open();
```

Comment:
can we make it more abstract? It would be great to have a universal plugin for showing such notifies.

I mean its better to unhardcode texts and button handlers and pass them as options.

Reply:
I can do that but can you describe the mode of this notifier like `confirmation window`.

Reply:
```js
this.$notify({
  message: 'New version is available',
  type: 'confirm',
  buttonText: 'Refresh',
  okHandler(){ ... }
  //...
})
```

Reply:
okay but what about `what's new` button?